Entrega Miro Scevola

SabaM10 opened the pull request.

SabaM10 committed:
Entrega Miro Scevola

gbobr commented:
run tests

gbobr commented:
Can one of the admins verify this patch?

gbobr reviewed (approved):
Ejercicio 1 correcto
Falta entrega del ejercicio 2

SabaM10 committed:
Add files via upload

soy tontin y hetero............................................................................................................................................................................................................................ filico

SabaM10 committed:
Merge branch 'master' into master

SabaM10 commented:
run tests

gbobr commented:
run tests

SabaM10 committed:
Update main.c

SabaM10 commented on ejercicio2/main.c:
```diff
@@ -1,14 +1,24 @@
-#include<stdio.h>
```
aca lo agregue al stdio.h alabado sea duki

gbobr commented:
rerun tests

SabaM10 committed:
Update main.c

gbobr commented:
rerun tests

SabaM10 committed:
Ejercicio 3

gbobr commented:
run tests

gbobr reviewed (approved):
Ejercicio aprobado
